tools/validators/instance_validator/validate/generate_universe.py

```diff
@@ -23,23 +23,32 @@
 from yamlformat.validator import presubmit_validate_types_lib
 from yamlformat.validator import namespace_validator
 
-def BuildUniverse():
+def BuildUniverse(modified_types_filepath=None):
   """Generates the ontology universe.
 
+  Args:
+    modified_types_filepath: filepath to modified ontology types
+
   Returns:
     Generated universe object.
   """
 
-  ontology_validator_exists = path.exists(path.join(
-      '..', 'ontology_validator'))
-  ontology_exists = path.exists(path.join('..', '..', '..', 'ontology'))
+  if modified_types_filepath:
+    modified_ontology_exists = path.exists(modified_types_filepath)
```

Comment:
add the `expanduser` here to support relative file paths like in here: 

https://github.com/google/digitalbuildings/blob/master/tools/rdf_generator/rdfformat/rdf_generator.py#L37

Reply:
Done

Reply:
responded offline